content/docs/gto/get-started.md

```diff
@@ -0,0 +1,162 @@
+---
+description:
+  'Learn how you can use GTO to create Artifact Registry in Git repository'
+---
+
+# Get Started
+
+GTO helps you build a Artifact Registry out of your Git repository. It does so
+by creating Git tags of [special format](/doc/gto/user-guide) and managing
+[`artifacts.yaml` metafile](/doc/gto/user-guide). Since committing large files
```

Comment:
Instead of a big User Guide, for now we can have a single guide page explaining these formats and their mechanics (again, similar to DVCLive's [Folder Structure](https://dvc.org/doc/dvclive/outputs) doc).

Reply:
Thanks for the all your feedback. We were releasing MLEM 0.3.0, so I was a bit off this. In general:
1. I processed your feedback - thanks! - feel free to bring more
2. Let's focus on GS, but I'll work on other things while I wait for you
3. Let's see if UG can fit a single page. Don't want to complicate things and write extra things, but it may be required to split in subpages.

Reply:
Sounds good. Not sure I understood what you're expecting from me (waiting) though. Thanks

Reply:
Np. Just keep reviewing please 😂

Reply:
K 🙂